packages/truffle-debugger/lib/data/sagas/index.js

```diff
@@ -10,6 +10,7 @@ import { TICK } from "lib/trace/actions";
 import * as actions from "../actions";
 
 import data from "../selectors";
+import solidity from "../../solidity/selectors";
```

Comment:
You can do `from "lib/solidity/selectors"` instead of the relative path. (Not using relative paths makes it easier to rename files, etc.)

Reply:
OK, changed.